tools/addrundata/rundata.go

```diff
@@ -103,22 +102,12 @@ func parseInit(sc *bufio.Scanner) (*mpb.Metadata, error) {
 	return nil, errors.New("func init() was not terminated")
 }
 
-var tmpl = template.Must(template.New("metadata.textproto").Parse(
-	`# proto-file: proto/metadata.proto
-# proto-message: Metadata
-
-uuid: "{{.UUID}}"
-plan_id: "{{.PlanID}}"
-description: "{{.Description}}"
-`))
-
 // writeProto generates a complete metadata.textproto to the writer.
 func writeProto(w io.Writer, md *mpb.Metadata) error {
-	return tmpl.Execute(w, struct {
-		UUID, PlanID, Description string
-	}{
-		UUID:        md.Uuid,
-		PlanID:      md.PlanId,
-		Description: md.Description,
-	})
+	bytes, err := prototext.Marshal(md)
+	if err != nil {
+		return err
+	}
+	_, err = w.Write(bytes)
```

Comment:
Write the proto-file and proto-message also?

Reply:
Done